src/controllers/EventController.ts

```diff
@@ -75,4 +81,38 @@ export class EventController {
 
     return this.eventMapper.entityToResponse(deletedEvent);
   }
+
+  @Post('/:eventID/signin')
+  @ResponseSchema(Attendance)
```

Comment:
probably can drop since this returns null

Reply:
I can make an AttendanceMapper and stuff and then have a proper AttendanceResponsePayload or smth like that. And yeah I can also just do what you said here